config.ini

```diff
@@ -1,24 +1,24 @@
-[Server]
```

Comment:
What changed in this file. I suspect it was something to do with line endings, so let's make sure it was supposed to change.